src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx

```diff
@@ -260,7 +260,9 @@ export default class MutationAssessorColumnFormatter
         );
 
         // add tooltip for valid values
-        if (text.length > 0 && text !== NA)
+        if (text.length > 0 &&
+            text.toLowerCase() !== "na" &&
+            text.toLowerCase().indexOf("not available") === -1)
```

Comment:
There's a whole bunch of them here: https://github.com/cbioportal/cbioportal-frontend/blob/9cd67c047369eeeaf1ec7d7fee13871e8165cc70/src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js#L16-L33. Might be some duplication in importer/backend as well. Main problem is that there are a ton of these values in the db, but those studies have never been reimported, so the improved importer never fixed them. Prolly good to have similar checks in frontend